src/resources/elements/primeDesignSystem/pform-input/pform-input.scss

```diff
@@ -1,6 +1,7 @@
 @import "../styles";
 
 pform-input {
+  display: block;
```

Comment:
This change is already introduced in another PR https://github.com/PrimeDAO/prime-deals-dapp/pull/80 and after it is merge I will rebase